ansible/roles/automation/nightly_pr/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: schedule opening nightly PRs
+  cron:
+    name: "openclose_{{ item.name }}"
```

Comment:
Here can be something like: ``` "openclose_{{ flow }}-{{ item.name }}"```

Reply:
flow is just another variable such as hour, minute, etc.. that defines the template to be used, for freeipa default is "ci", for pki nightly-based is "pki".